CI/wininstaller/installer.iss

```diff
@@ -0,0 +1,733 @@
+#define VCMITeam "VCMI Team"
+#define VCMICopyright "Copyright © VCMI Community. All rights reserved."
+
+#define VCMIHome "https://vcmi.eu/"
+#define VCMIContact "https://discord.gg/chBT42V"
+
+[Setup]
+AppId=VCMI
+AppName=VCMI
+AppVersion={#AppVersion}.{#AppBuild}
+AppVerName=VCMI
+AppPublisher={#VCMITeam}
+AppPublisherURL={#VCMIHome}
+AppSupportURL={#VCMIContact}
+AppComments={#AppComment}
+DefaultDirName={code:GetDefaultDir}
+DefaultGroupName=VCMI
+UninstallDisplayIcon={app}\VCMI_launcher.exe
+OutputBaseFilename={#InstallerName}_{#InstallerArch}_{#AppVersion}.{#AppBuild}
+PrivilegesRequiredOverridesAllowed=commandline dialog
+ShowLanguageDialog=yes
+DisableWelcomePage=no
+DisableProgramGroupPage=yes
+ChangesAssociations=yes
+UsePreviousLanguage=yes
+DirExistsWarning=no
+UsePreviousAppDir=yes
+UsePreviousTasks=yes
+UsePreviousGroup=yes
+DisableStartupPrompt=yes
+UsedUserAreasWarning=no
+WindowResizable=no
+CloseApplicationsFilter=*.exe
+Compression=lzma2/ultra64
+SolidCompression=yes
+ArchitecturesAllowed={#InstallerArch}compatible
+LicenseFile={#LicenseFile}
+SetupIconFile={#IconFile}
+WizardSmallImageFile={#SmallLogo}
+WizardImageFile={#WizardLogo}
+
+; Version informations
+MinVersion=6.1sp1
+VersionInfoCompany={#VCMITeam}
+VersionInfoDescription=VCMI {#AppVersion} Setup (Build {#AppBuild})
+VersionInfoProductName=VCMI
+VersionInfoCopyright={#VCMICopyright}
+VersionInfoVersion={#AppVersion} 
+VersionInfoOriginalFileName={#InstallerName}.exe
+
+[Languages]
+Name: "english"; MessagesFile: "{#LangPath}\English.isl"
+Name: "czech"; MessagesFile: "{#LangPath}\Czech.isl"
+Name: "chinese"; MessagesFile: "{#LangPath}\ChineseSimplified.isl"
+Name: "finnish"; MessagesFile: "{#LangPath}\Finnish.isl"
+Name: "french"; MessagesFile: "{#LangPath}\French.isl"
+Name: "german"; MessagesFile: "{#LangPath}\German.isl"
+Name: "hungarian"; MessagesFile: "{#LangPath}\Hungarian.isl"
+Name: "italian"; MessagesFile: "{#LangPath}\Italian.isl"
+Name: "korean"; MessagesFile: "{#LangPath}\Korean.isl"
+Name: "polish"; MessagesFile: "{#LangPath}\Polish.isl"
+Name: "portuguese"; MessagesFile: "{#LangPath}\BrazilianPortuguese.isl"
+Name: "russian"; MessagesFile: "{#LangPath}\Russian.isl"
+Name: "spanish"; MessagesFile: "{#LangPath}\Spanish.isl"
+Name: "swedish"; MessagesFile: "{#LangPath}\Swedish.isl"
+Name: "turkish"; MessagesFile: "{#LangPath}\Turkish.isl"
+Name: "ukrainian"; MessagesFile: "{#LangPath}\Ukrainian.isl"
+Name: "vietnamese"; MessagesFile: "{#LangPath}\Vietnamese.isl"
+
+[Files]
+Source: "{#SourceFilesPath}\*"; DestDir: "{app}"; Flags: ignoreversion recursesubdirs createallsubdirs; BeforeInstall: PerformHeroes3FileCopy
+
+[Icons]
+Name: "{group}\{cm:ShortcutLauncher}"; Filename: "{app}\VCMI_launcher.exe"; Comment: "{cm:ShortcutLauncherComment}"
+Name: "{group}\{cm:ShortcutMapEditor}"; Filename: "{app}\VCMI_mapeditor.exe"; Comment: "{cm:ShortcutMapEditorComment}"
```

Comment:
maybe also add Client, but only to start menu? The editor could also go only there, most users don't need to have it on desktop.

Reply:
I did, but removed it as client runs only with console window. That could be confusing for normal users.

Reply:
normal users will just run launcher from desktop :)

Reply:
Yes, and then non power users who run search in start menu / taskbar will be confused with just another icon which is likely for modders only.

Reply:
not insisting on that